tests/ts_cluster_riak_shell_basic_sql.erl

```diff
@@ -109,22 +106,24 @@ query_table_test(Pid, Conn) ->
     Pid ! Result.
 
 %% Stolen from the innards of riak_shell
+format_table({"", _}) ->
+    "";
+format_table({[[]], _}) ->
+    "";
+format_table({String, _}) ->
+    lists:reverse(tl(lists:reverse(lists:flatten(String)))).
```

Comment:
Concerned with trailing new-line characters? There's an app for that: `lib:nonl/1`.